dhis-2/dhis-support/dhis-support-commons/src/main/java/org/hisp/dhis/commons/jackson/jsonpatch/operations/RemoveByIdOperation.java

```diff
@@ -66,7 +66,7 @@ public JsonNode apply( JsonNode node )
 
         if ( !nodePathExists( node ) )
         {
-            return node;
+            throw new JsonPatchException( String.format( "Invalid path %s", path ) );
```

Comment:
Maybe add some tests to both Remove operations? Just so we our covered if anyone changes this behavior again

Reply:
@mortenoh sorry, this change supposed to be in another PR, reverted now.